### PR TITLE
Fix/instructions

### PR DESCRIPTION
**Pull request by aubeOSG**
- Removed some of the requirements of the controls tasks to reducing the time need to complete the assessment.
- Changed some language and fixed some grammar and spelling mistakes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
-> This assessment is based in nodejs, so you will need to have that installed along with npm.
+> This assessment is based in NodeJS, so you will need to have that installed along with NPM.
 
-Once you have your repo created, you will need to install some dependencies.
+Once you have your repository created, you will need to install some dependencies.
 
 `npm install`
 
-> One the dependencies is [parcelJS](https://parceljs.org/) (you don't need an understanding of parcel for this assessment) you may need to install this package globally.
+> One the dependencies is [parcelJS](https://parceljs.org/) (you don't need an understanding of parcel for this assessment) you may need to install this package globally. `npm install parcel -D -g`
```
Comment (CiceroFonsecaOSG) [inline README.md:19]: Can we move it up above the `npm install`? 

This way the candidate can install parcelJS before trying to run `npm install` and get an error, for sure there will be someone who will copy/paste/run and then read the parcelJS instruction 😅

File: README.md [hunk 2]
```diff
@@ -37,40 +36,30 @@ Using a framework of your choice (React, Vue, Angular, etc), create a single pag
...
-
-**Order**
-* Ascending (a to z / oldest to latest / low to high) 
-* Descending (z to a / latest to oldest / high to low) [Default]
-
-
-#### Completed
-A checkbox only made available to the User Courses page. When checked, only courses that have been completed will be displayed. Additionally, the completed field needs to be added to the Sort By control as an option only when on the User Course page.
\ No newline at end of file
+A dropdown menu containing the options below which are fields that can be sorted on and radio button group indicating sort order.
```
Comment (CiceroFonsecaOSG) [inline README.md:53]: I'm kind of confused here. 

This should be a dropdown with two groups, the first group is the sort option (title, created) and the second one the sort order (asc, desc), right? The `low to high` and `high to low` will not be used once we got rid of the number sort options, is this correct?

**Review by CiceroFonsecaOSG — CHANGES_REQUESTED** [review]
Small feedback @aubeOSG, other than that, I think this is in pretty good shape 👌

**Review by SezarGantousOSG — APPROVED** [review]
Other than what @CiceroFonsecaOSG is suggesting I think you did make it better and easier to ready through. I think the tables you added helps quite a bit.

**Pull request merged**